doc/manuals/user/filtering.md

```diff
@@ -188,7 +188,26 @@ described in detail in [the following section](geolocation.md#geo-located-querie
 
 [Top](#top)
 
-### String filter
+### String filters
+
+Two different types of string filters are supported:
+* filtering over attributes (scope type: FIWARE::StringQuery) and
```

Comment:
Looking to the style in other places of this file, filter types are surrounded by quotes. See line 186, for instance.

(Also to fix in the next line)